### PR TITLE
docs: Update internal rasa x url when using Rasa X/Enterprise deployed in the same namespace

### PR DESCRIPTION
**Pull request by HotThoughts**
`s/rasa-rasa-x/rasa-x-rasa-x/g`

CI Updates:
- Don't lint if the changes are inside `README.md` only.

### CODE REVIEW

File: examples/rasa/README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Therefore you would add the following to rasa-values.yaml:
 ## Extra environment variables used in the Rasa X/Enterprise configuration
 extraEnv:
  - name: RASA_MODEL_SERVER
-   value: http://example.com/api/projects/default/models/tags/production
+   value: http://rasa-x-rasa-x:5002/api/projects/default/models/tags/production
```
Comment (tczekajlo) [inline examples/rasa/README.md:74]: Here I'd leave it as it was. Rasa Model Server can point to any deployment, not necessarily within the same cluster, or namespace.

**Review by tczekajlo — CHANGES_REQUESTED** [review]

File: examples/rasa/README.md (hunk 1 above)
Comment (HotThoughts) [reply examples/rasa/README.md:74]: Doesn't it make sense to change it considering this file is titled **Using Rasa X/Enterprise deployed in the same namespace as a configuration endpoint**? 🤔

Comment (tczekajlo) [reply examples/rasa/README.md:74]: Ah, ok, I didn't see the title, then all good.

**Review by tczekajlo — APPROVED** [review]
LGTM

**Pull request merged**